smart-contracts/test/Lock/initializers.js

```diff
@@ -18,7 +18,7 @@ contract('Lock / initializers', accounts => {
 
   it('There are exactly 3 public initializers in PublicLock', async () => {
     const count = publicLockContract.abi.filter(x =>
-      x.name.toLowerCase().includes('initialize')
+      (x.name || '').toLowerCase().includes('initialize')
```

Comment:
This is just to catch the fallback func right?

Reply:
Yea cause `.name` is undefined for the fallback... so it broke the test. This seemed like an easy way to correct for that.